src/Adapter/Presenter/Product/ProductLazyArray.php

```diff
@@ -672,6 +672,10 @@ private function addPriceInformation(
         } else {
             $this->product['unit_price'] = $this->product['unit_price_full'] = '';
         }
+
+        Hook::exec('overrideProductPrices', array(
```

Comment:
```suggestion
        Hook::exec('actionAddPriceInformationAfter', array(
```
It's more relevant I think :thinking:

Reply:
Yes maybe, I took example on hook"overrideMinimalPurchasePrice" to name mine.

Reply:
Wdty @Quetzacoalt91 ? (He's our expert :sunglasses: )